commons-mongo/src/test/scala/com/avsystem/commons/mongo/Something.scala

```diff
@@ -0,0 +1,95 @@
+package com.avsystem.commons
+package mongo
+
+import com.avsystem.commons.serialization.GenCodec
+
+import scala.util.Random
+import scala.util.hashing.MurmurHash3
+
+object Randoms {
```

Comment:
I guess `scalacheck` is not invented here :)

Reply:
See above, 10/10 would use `scalacheck`.